terraform/application/variables.tf

```diff
@@ -74,3 +74,8 @@ variable "uploads_storage_account_name" {
   type    = string
   default = null
 }
+
+variable "enable_logit" {
```

Comment:
Could we keep the variables in this file in alphabetical order, just to make them easier to find, and same for the `tfvars` files.

Reply:
fixed as requested